src/main/java/net/openhft/chronicle/core/threads/Timer.java

```diff
@@ -64,13 +64,13 @@ public void schedule(@NotNull Runnable eventHandler, long initialDelayMs) {
         }, initialDelayMs, 0));
     }
 
-    private static class ScheduledEventHandler implements EventHandler, Closeable {
+    private static final class ScheduledEventHandler implements EventHandler, Closeable {
 
         @NotNull
         private final VanillaEventHandler eventHandler;
         private final long initialDelayMs;
         private final long periodMs;
-        private transient volatile boolean closed;
+        private volatile boolean closed;
```

Comment:
why?

Reply:
Timer is `final` and does not implement `Serializable`. So, there is never a use for `transient`

Reply:
yep - why ?

Reply:
'transient' means a field will not participate in serialization. But the class itself is not serializable and cannot be subclassed. So it does not make sense

Reply:
OK